SparkPost/2web2ui
# EO-878 Add slack and webhook notification icons

### What Changed
 - Added the notification icons
 - Changed the email address form name from `email_addresses` to `emails`.
 - Added the option to allow users to enter emails using newline and commas

### How To Test
 - Enable user account option `alerts`. This is already enabled in appteam staging.
 - See [here](https://github.com/SparkPost/2web2ui/blob/master/docs/feature-flags.md) to enable user options. Alternatively, [point your local upstreams to staging](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams) and login to our appteam, CID 107 account
 - Edit your openresty configs to contain 
```
    location /api/v1/alerts {
      proxy_pass http://sp-alerts-api;
      include cors;
    }
```
under `server:80`,   `location /api/v1`
 - Navigate to `/alerts-new`.
 - Click on the create alert button.
 - Verify that the new notification channels show up and can be created. 
 - Create one and then click on the one you just created to navigate to the details page.
 - Verify that the new notification channels are displayed.
